trino/trino.go

```diff
@@ -733,10 +737,11 @@ func (st *driverStmt) QueryContext(ctx context.Context, args []driver.NamedValue
 
 func (st *driverStmt) exec(ctx context.Context, args []driver.NamedValue) (*stmtResponse, error) {
 	query := st.query
-	var hs http.Header
+	hs := make(http.Header)
+	// Ensure we have full date precision support.
```

Comment:
I checked [the docs](https://trino.io/docs/current/develop/client-protocol.html) to see if we could copy the description but it doesn't mention this header so we can't reference it here. How about this:
```suggestion
	// Ensure the server returns timestamps preserving their precision, without truncating them to timestamp(3).
```